python/CUDAQuantumExtension.cpp

```diff
@@ -33,10 +34,11 @@
 #include "runtime/mlir/py_register_dialects.h"
 #include "utils/LinkedLibraryHolder.h"
 #include "utils/OpaqueArguments.h"
-#include "mlir/Target/LLVMIR/Dialect/LLVMIR/LLVMToLLVMIRTranslation.h"
 
-#include "cudaq/Optimizer/CAPI/Dialects.h"
+#include "../runtime/cudaq/platform/orca/orca_qpu.h"
+
 #include "mlir/Bindings/Python/PybindAdaptors.h"
+#include "mlir/Target/LLVMIR/Dialect/LLVMIR/LLVMToLLVMIRTranslation.h"
```

Comment:
Is this new include (and the one on line 16) still needed? I looked at `runtime/cudaq/platform/orca/orca_qpu.h`, and I can't figure out why they would be new dependencies as a result of these changes. (It's to leave it if it is needed ... I just figured I'd double check just in case they were leftovers from an earlier WIP?)

Reply:
Thanks, I will confirm this...

Reply:
Based on my quick testing, it looks like mlir/Target/LLVMIR/Dialect/LLVMIR/LLVMToLLVMIRTranslation.h may be required but "cudaq/Optimizer/CAPI/Dialects.h" is not.

Reply:
Done. 8e6e94aca7e98ad31e16ba62286bb1d86c81cda9